jni/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheImpl.java

```diff
@@ -626,5 +632,34 @@ public ServerStatistics stats() {
         return stats;
     }
 
+    @Override
+    public <T> T execute(String scriptName, Map<String, ?> params) {
+        MapType argsMap = new MapType();
+        JBossMarshaller jb = new JBossMarshaller();
+        for (String parName : params.keySet()) {
+            ByteBuffer b;
+            try {
+                b= jb.objectToBuffer(params.get(parName));
```

Comment:
nit: space after the `b`. I just use the same IDE formatting rules as Infinispan Java code to fix these.